.changes/next-release/removal-AWSSDKforJavav2-7557e89.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "AWS SDK for Java v2", 
+    "type": "removal", 
+    "description": "remove deprecated class 'software.amazon.awssdk.core.util.StringUtils'"
+}
```

Comment:
SDK users probably don't care about this change, so we probably don't need a changelog entry.